packages/@okta/vuepress-site/docs/guides/embedded-siw/main/index.md

```diff
@@ -0,0 +1,428 @@
+    ```
+
+#### Run the sample application
+
+1. In your sample directory, `simple-spa`, you can run a static site web server using the following command:
+
+    ```bash
+    npx http-server . -p 3000
+    ```
+
+    >**Note:** If not installed previously, you are prompted to install the `npx` package.
+
+1. Navigate to `http://localhost:3000`. The simple web page appears with a message that you're not signed in and displays the Sign-In Widget.
+
+1. Sign in with a user from your org assigned to the app integration. The simple web page appears with the signed-in user's email address.
+
+### Sign in to Okta with the default dashboard
+
+In this case, you use the Widget to sign in to the default Okta dashboard. This requires taking the Widget initialization code and modifying the success behavior to redirect to your org's dashboard.
+
+```javascript
+function success(res) {
+  if (res.status === 'SUCCESS') {
+    res.session.setCookieAndRedirect('https://${yourOktaDomain}/app/UserHome');
+  }
+}
+```
+
+### Sign in to Okta and SSO directly to an app
+
+If you want to sign the user directly in to an application within Okta, you redirect to the specific URL for that application. To find that URL, go to that application's page in your Okta org and find [the embed link](https://help.okta.com/okta_help.htm?id=ext-apps-page-show-application-embed-links).
+
+### Sign in to Okta with a custom dashboard
+
+If you are signing your users in to Okta, but you don't want to use the Okta dashboard, then you can change the redirect URL to point to your custom portal instead.
+
+```javascript
+function success(res) {
+  if (res.status === 'SUCCESS') {
+    res.session.setCookieAndRedirect('https://example.com/dashboard');
+  }
+}
+```
+
+### Sign in to your application
+
+If want to use the Widget to sign in to your own application instead of Okta, you need to [set up a Custom Authorization Server](/docs/guides/customize-authz-server/) in Okta.
+
+### Server-side web application using Authorization Code flow
+
+```javascript
+
+const signIn = new OktaSignIn({
+  baseUrl: 'https://${yourOktaDomain}',
+  el: '#widget-container',
+  clientId: '${clientId}',
+  // must be in the list of redirect URIs enabled for the OIDC app
+  redirectUri: '${redirectUri}',
+  authParams: {
+    issuer: 'https://${yourOktaDomain}/oauth2/default',
+    pkce: false,
+    responseType: ['code']
+  }
+});
+
+// A query parameter named `code` is passed to the login redirect URI
+// This should be handled by server-side code. The code can be exchanged for tokens
+signIn.showSignInAndRedirect();
+
+```
+
+### SPA or Native application using PKCE
+
+```javascript
+
+const signIn = new OktaSignIn({
+  baseUrl: 'https://${yourOktaDomain}',
+  el: '#widget-container',
+  clientId: '${clientId}',
+  // must be in the list of redirect URIs enabled for the OIDC app
+  redirectUri: '${redirectUri}',
+  authParams: {
+    issuer: 'https://${yourOktaDomain}/oauth2/default'
+  }
+});
```

Comment:
Please add `useInteractionCodeFlow: true` to turn on the OIE features in widget.